app/models.py

```diff
@@ -27,6 +27,11 @@ class MemberInput(BaseModel):
     graduated: bool
     phone: Optional[str]
 
+class MemeberUpdate(BaseModel):
+    realName: Optional[str]
```

Comment:
Shall these be snake case?

Reply:
Actually, `realName` is kind of an old token. We could probably change this fieldname in its entirety as it really doesn't make sense in this context.